### DISCUSSION TITLE
How to keep the size of the sprite unchanged when the container is scaled？

### DISCUSSION THREAD

**Discussion by bibiehy** [pixijs/pixijs#8547] [Q&A]
pixi@6.5.1

https://codesandbox.io/s/epic-bouman-o71tef?file=/src/index.js

How to keep the sprite size unchanged when the mouse wheel is scrolled.

Just like when the map is zoomed, the building marker icon does not change.

**Comment by ivanpopelyshev** [comment]
https://github.com/pixijs/pixijs/wiki/v5-Hacks#ignore-parent-scale--rotation
